scheduler/src/cook/rest/api.clj

```diff
@@ -730,7 +738,14 @@
                                                                 (str/lower-case operator))
                                   :constraint/pattern pattern}]))
                             constraints)
-        container (if (nil? container) [] (build-container user db-id container))
+        pool-name (or (:pool/name pool) (config/default-pool))
+        default-container-for-pool (get-in config/config [:settings :pools :default-container-for-pool])
```

Comment:
Given the new location and structure, maybe `:default-container-for-pool` isn't the best name -- maybe just `:default-containers`, since there can be multiple and we're already under the `:pools` key.

Reply:
Changed.